clone/README.md

```diff
@@ -18,40 +34,32 @@ MariaDB [(none)]> GRANT ALL PRIVILEGES ON google_xenon2022.* TO 'clonetool'@loca
 MariaDB [(none)]> FLUSH PRIVILEGES;
 ```
 
-Now you can clone the log with:
-
-```
-go run ./clone/cmd/ctclone --alsologtostderr --v=1 --log_url https://ct.googleapis.com/logs/xenon2022/ --mysql_uri 'clonetool:letmein@tcp(localhost)/google_xenon2022'
-```
+## Tuning
 
-See the optional flags in the `ctclone` tool to configure tuning parameters.
-## Docker
-
-To build a docker image, run the following from the `trillian-examples` root directory:
+The clone library logs information as it runs using `glog`.
+Providing `--alsologtostderr` is provided to any tool using the library, you should see output such as the following during the cloning process:
```

Comment:
`is provided` -> `is passed` (repetition)

Reply:
Ta.